frontend/src/views/DiagnosticTunnel/DiversificationMeasureSteps/index.vue

```diff
@@ -1,81 +1,81 @@
 <template>
-  <fieldset v-if="stepUrlSlug === 'menu'">
-    <legend class="text-left my-3">
-      J'ai mis en place un menu végétarien dans ma cantine :
-      <span class="fr-hint-text mt-2">Optionnel</span>
-    </legend>
-    <v-radio-group class="my-0" v-model="payload.vegetarianWeeklyRecurrence" hide-details @change="updatePayload">
-      <v-radio v-for="item in frequency" :key="item.value" :label="item.label" :value="item.value"></v-radio>
-    </v-radio-group>
-  </fieldset>
-  <fieldset v-else-if="stepUrlSlug === 'options'">
-    <legend class="text-left my-3">
-      Le menu végétarien proposé est :
-      <span class="fr-hint-text mt-2">Optionnel</span>
-    </legend>
-    <v-radio-group class="my-0" v-model="payload.vegetarianMenuType" hide-details @change="updatePayload">
-      <v-radio v-for="item in menuTypes" :key="item.value" :label="item.label" :value="item.value"></v-radio>
-    </v-radio-group>
-  </fieldset>
-  <fieldset v-else-if="stepUrlSlug === 'composition'">
-    <legend class="text-left mb-2 mt-3">
-      Le plat principal de mon menu végétarien est majoritairement à base de :
-      <span class="fr-hint-text mt-2">Optionnel</span>
-    </legend>
-    <v-checkbox
-      hide-details="auto"
-      class="mt-2"
-      v-model="payload.vegetarianMenuBases"
-      :multiple="true"
-      v-for="item in menuBases"
-      :key="item.value"
-      :value="item.value"
-      :label="item.label"
-      @change="updatePayload"
-    />
-  </fieldset>
-  <div v-else-if="stepUrlSlug === 'plan'">
-    <fieldset>
+  <v-form @submit.prevent>
+    <fieldset v-if="stepUrlSlug === 'menu'">
       <legend class="text-left my-3">
-        J'ai mis en place un plan pluriannuel de diversification des protéines incluant des alternatives à base de
-        protéines végétales
+        J'ai mis en place un menu végétarien dans ma cantine :
+        <span class="fr-hint-text mt-2">Optionnel</span>
```

Comment:
Note to self: Je ne comprends pas pourquoi je vois ces modifs qui ne sont pas dans la branche de la PR